ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424__model_serving_cluster_setting.robot

```diff
@@ -0,0 +1,99 @@
+    ${status_single_mode}=  Get Checkbox State Of Single Modelserving platforms
+    IF  "${status_single_mode}"=="True"
+        Unselect Checkbox   ${SINGLE_MODE_SERVING_CHECK_BOX}
+    END
+
+    ${status_multi_mode}=  Get Checkbox State Of Multi Modelserving platforms
+    IF  "${status_multi_mode}"=="True"
+        Unselect Checkbox   ${MULTI_MODE_SERVING_CHECK_BOX}
+    END
+    Capture Page Screenshot
+    ${status} =   Evaluate    '${status_single_mode}' == 'True' or '${status_multi_mode}' == 'True'
+    Run Keyword If    '${status}' == 'True'    Save Changes In Cluster Settings
+
+Sute Tierdown
+    [Documentation]    Delete DS project and select both model serving options
+    Delete Data Science Project                 ${project_tittel}
+    Wait Until Data Science Project Is Deleted  ${project_tittel}
+    Open Dashboard Settings    settings_page=Cluster settings
+    Reload Page
+    Wait Until Page Contains Element    //input[@id="multi-model-serving-platform-enabled-checkbox"]  timeout=20
+    Select Both Model Serving Platforms
+    Close Browser
+
+*** Test Cases ***
+Verify Correct Value in DS Project after Enabling Both Model Serving Platforms
+    [Documentation]    Verifies that correct values are present in the DS project after enabling both model serving platforms
+    #Case 1
+    Open Dashboard Settings    settings_page=Cluster settings
+    Reload Page
+    Wait Until Page Contains Element    //input[@id="multi-model-serving-platform-enabled-checkbox"]  timeout=20
+    Select CheckBox Multi Model Serving Platforms
+    Select CheckBox Single Model Serving Platforms
+    Capture Page Screenshot
+    Save Changes In Cluster Settings
+    Open Data Science Project Details Page      ${project_tittel}
+    Click Element    //*[contains(@class, 'pf-v5-c-jump-links') and contains(@class, 'pf-v5-c-jump-links__link-text') and contains(., 'Models and model servers')]
+    Wait Until Page Contains Element    //*[contains(text(), "Deploy model")]
+    Wait Until Page Contains Element    //*[contains(text(), "Add model server")]
+
+Verify Correct Value in DS Project after Enabling Multi Model Serving Platforms
+    [Documentation]    Verifies that correct values are present in the DS project after enabling both Multi Model serving platforms
+    #Case 2
+    Open Dashboard Settings    settings_page=Cluster settings
+    Reload Page
+    Wait Until Page Contains Element    //input[@id="multi-model-serving-platform-enabled-checkbox"]  timeout=20
+    Select CheckBox Multi Model Serving Platforms
+    Capture Page Screenshot
+    Save Changes In Cluster Settings
+    Open Data Science Project Details Page      ${project_tittel}
+    Click Element    //*[contains(@class, 'pf-v5-c-jump-links') and contains(@class, 'pf-v5-c-jump-links__link-text') and contains(., 'Models and model servers')]
+    Wait Until Page Contains Element     //button[contains(text(), 'Add model server')]
```

Comment:
this is not enough. We must check that the only Multi model platform card/buttons are present and Single model ones are not

Reply:
could you add the checks like in the previous test about cards?
e.g.
    Wait Until Page Contains Element    //*[@id="single-serving-platform-card"]
    Wait Until Page Does Not Contain Element    //*[@id="multi-serving-platform-card"]